### DISCUSSION TITLE
Bootstrap v5+ does not work with lenis.

### DISCUSSION THREAD

**Discussion by DarkRL** [darkroomengineering/lenis#255] [General]
I needed a good smooth mouse scrolling plugin for my website. After having tried many they all seem to break and I initially thought it was the plugin itself. But after a few hours of head scratching and debugging I found out that the Bootstrap stylesheet (v5+) actually breaks the lenis plugin. The scrolling completely breaks and it took me too long to find out.

Great plugin btw, nothing wrong with it! Just wanted to warn that the new bootstrap version doesn't seem to be compatible with Lenis, however any version below 5 seem to work great. Thanks for making such an awesome plugin free!

**Comment by clementroche** [comment]
can you provide a minimal reproduction link ?

**Comment by DarkRL** [comment]
Hi @clementroche, I honestly forgot I still had this question open but I have fixed it.

From Bootstrap v5.0+ its stylesheet defines:
```
:root {
    scroll-behavior: smooth
}
```

So I can simply override it in my own stylesheet with:
```
:root {
    scroll-behavior: auto !important
}
```
